django_prices/models.py

```diff
@@ -70,20 +72,53 @@ def get_db_prep_save(self, value, connection):
     def value_to_string(self, obj):
         value = self.value_from_object(obj)
         if value is not None:
-            return value.net
-        return super(PriceField, self).value_to_string(value)
+            return value.value
+        return super(AmountField, self).value_to_string(value)
 
     def formfield(self, **kwargs):
         defaults = {'currency': self.currency,
-                    'form_class': forms.PriceField}
+                    'form_class': forms.AmountField}
         defaults.update(kwargs)
-        return super(PriceField, self).formfield(**defaults)
+        return super(AmountField, self).formfield(**defaults)
 
     def get_default(self):
-        default = super(PriceField, self).get_default()
+        default = super(AmountField, self).get_default()
         return self.to_python(default)
 
     def deconstruct(self):
-        name, path, args, kwargs = super(PriceField, self).deconstruct()
+        name, path, args, kwargs = super(AmountField, self).deconstruct()
         kwargs['currency'] = self.currency
         return name, path, args, kwargs
+
+
+class PriceField(object):
+
+    description = 'A field that stores a price.'
+
+    def __init__(self, net_field='price_net', gross_field='price_gross',
+                 verbose_name=None, **kwargs):
+        self.net_field = net_field
+        self.gross_field = gross_field
+
+    def __str__(self):
+        return ('PriceField(net_field=%s, gross_field=%s)' %
+                (self.net_field, self.gross_field))
+
+    def __get__(self, instance, cls=None):
+        if instance is None:
+            return self
+
+        net_val = getattr(instance, self.net_field)
+        gross_val = getattr(instance, self.gross_field)
+
```

Comment:
Functions in this module are rather simple, I'm not sure if these blank lines add much to readability.